article-analysis-project/article-processing.js

```diff
@@ -0,0 +1,44 @@
+//Modules
+const unfluff = require('unfluff');
+const cheerio = require('cheerio');
+const request = require('request');
+//Variables and Consts
+const url = 'https://abcnews.go.com/Politics/wireStory/canceling-ivanka-trump-spurs-backlash-kansas-college-71174430?cid=clicksource_4380645_2_heads_hero_live_headlines_hed'
+
+//Main
+
+request(url, (error, response, html) => {
+    if (!error && response.statusCode == 200){
+        console.log(html);
+        data = unfluff(html);
+        console.log(data.text);
+        console.log(data.author)
+        author = data.author
+        console.log(occurrences(data.text, " he ", false))
+    }
+
+})
+
+function occurrences(string, subString, allowOverlapping) {
+
+    string += "";
```

Comment:
Two quick notes: 

1. `n.toString()` is a more clear way to convert a number to a string
2. It's good practice to NOT modify passed parameters
3. `string` and `substring` are not great names because I don't know what they mean necessarily.  I would consider redefining this function as:
4. allowOverlapping seems like it should be optional; you can set default values for parameters by using `={insert_default_here}`.

`function countOccurences(haystack, needle, allowOverlapping=true)`

Also for what it is worth I would normally put this kind of generic utility method in a `utils.js` that gets unit tested.